content/en/docs/ops/configuration/traffic-management/multicluster/index.md

```diff
@@ -16,6 +16,40 @@ load-balanced across all clusters in the mesh for a given service.
 
 If there is no case where this traffic should be sent across clusters, consider giving the services a different name in each cluster.
 
+### Locality Load Balancing
+
+When using a multi-cluster mesh for redundancy and resiliency, it may be desirable to only send cross-cluster traffic
+when necessary. Using [Locality Load Balancing](/docs/tasks/traffic-management/locality-load-balancing/) settings, you
+can configure locality weighted distribution or failover.
+
+On top of default locality components of `region/zone/subzone`, you can use [`failoverPriority`](/docs/reference/config/networking/destination-rule/#LocalityLoadBalancerSetting)
+to failover based on the cluster or network explicitly. Using the following destination rule, traffic will prefer first
+in-cluster traffic, then in-network, then in-region, etc.
```

Comment:
Not exactly, matching all the labels first, then second, match the first N-1 labels.